draft-ietf-rats-corim.md

```diff
@@ -1216,15 +1216,21 @@ matched.
 {::include cddl/stateful-environment-record.cddl}
 ~~~
 
-The series object is an array of `conditional-series-record` that has both Reference and Endorsed Values.
-Each `conditional-series-record` record is evaluated in the order it appears in the series array.
-The Endorsed Values are accepted if the series condition in a `conditional-series-record` matches the ACS.
-The first `conditional-series-record` that successfully matches an ACS Entry terminates the matching and the corresponding Endorsed Values are accepted.
-If none of the series conditions match an ACS Entry, the triple is not matched,
-and no Endorsed values are accepted.
+The series is an array of `conditional-series-record` that has a `selection`, and an `addition`, both expressed as a list of `measurement-map`.
+The `selection` and `addition` operate within the scope of the conditional endorsement series triple record's matching `condition`.
 
-The `authorized-by` value in `measurement-map` in the stateful environment, if present,
-applies to all measurements in the triple, including `conditional-series-record` records.
+For each `conditional-series-record` entry, if the `selection` matches the chosen ACS entry, the `addition` is added to the ACS.
```

Comment:
> Addressed: Remove the chosen term!

The original text was trying to capture the detail that the selection scope is within the range of the `condition` and that the `addition` is within the same range.  This is a necessity since the series records don't have a qualifying environment-map.